src/main/java/Parser.java

```diff
@@ -0,0 +1,293 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import exceptions.EmptyTodoException;
+import exceptions.InvalidDeadlineException;
+import exceptions.InvalidEventException;
+import exceptions.InvalidNumberException;
+import exceptions.UnknownCommandException;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.ToDos;
+
+/**
+ * class consisting of static methods which are used to make sense of the user commands
+ */
+public class Parser {
+    /**
+     * Parse the user input and get the right response
```

Comment:
Is is better if an empty line can be placed to separate the description and @param?